libs/asset-viewer/src/lib/state/asset-search/asset-search.selector.ts

```diff
@@ -195,6 +196,26 @@ export const selectFilters = <T extends string>(
     }
   );
 
+export const selectWorkgroupFilters = createSelector(
```

Comment:
Other Filters contain a Filter even if the amount is 0. Currently only Workgroups are shown which do have at least 1 asset. Can we make the Behavior the same?